java/src/test/java/ai/rapids/cudf/ColumnVectorTest.java

```diff
@@ -6677,6 +6678,36 @@ void testApplyBooleanMaskFromListOfStructure() {
     }
   }
 
+  @Test
+  void testColumnViewWithNonEmptyNullsIsCleared() {
```

Comment:
could you add comments in this test on what it is doing? specifically what about this is a "non-empty-nulls?

Right now I know you are building a column, but I do not know the significance of say `Optional.of(2L)`. It would be great  to comment on anything important here.

Reply:
I have added comments. I hope this makes it clear what we are doing